content/docs/guide/gamemodes/climb.md

```diff
@@ -11,16 +11,92 @@ tags:
+
+**Duck Roll [1.6 Climb only]**: Sometimes called Silent Walking, Russian Walking, or G-strafe in Counter-Strike, Duck Roll is its given name in Half-life so for consistency's sake It will be called Duck Roll.  
+It is possible to do a "Bhop" sequence comprised entirely of Double Ducks. This lets you get pass the Bhop speed cap since it is not considered a jump. It is very hard to maintain Duck Rolls given how much more frequent you must duck compared to jumps in a Bhop sequence. Duck Rolls are vertically limited, so even with more speed they may not reach a place a Bhop can. DuckRoll's effectivness heavily depends on the map.  
 
-Climb works in the same way that bunny hopping does; pressing W or S combined with a strafe key causes the player to receive more velocity than simply holding W or S. Combining this with turning the player's camera in line with the strafe key, a player can get lots of momentum to jump to far platforms. This is normally called a **prestrafe**.
+**High Jump [1.6 Climb only]**: In Source and Goldsrc there exists a property called **edge friction** it decides what factor that "normal" friction will be multiplied by when a player is close to an edge. If it is set higher than 1, you will slow down every time you try to jump from the edge of a platform. These are the edge frictions for each Climb mode:  **TBD (MomClimb)**,     **1 (KZT)**,       **2 (1.6 Climb)**.  
+**[Notice that 1.6 Climb is the only Climb mode whose edge friction is diffrent than the normal friction]**.  
+You are considered to be next to an edge if both of these conditions are true:   
+**1. In the direction you are moving (not facing), an "invisible line" 16 units in front of the bottom-center of your collision box (your feet) does not intersect with anything solid**.  
+**3. From the end of that 16 unit invisible line, another 70 units (52 units if crouched) invisble line pointing down does not insersect with anything solid**.  
+[For refrenece, player height is 72 units]. 
+![image](../../../../assets/High_Jump.png)  
+you can still "trick" the game into thinking you are not next to an edge even when you are on a high up platform. Remember the engine only checks in the direction you are moving, so if you are moving in a direction that does not intersect with a platform's edge you are not considered "next to an edge" regardless of how close you actually are to it. We can use this to maintain a normal speed. High Jumps are typically done by moving perpendicular to the edge you want to jump from while sticking very close it, and then only turning towards the edge for your pre-strafe. This minimizes the amount of time you are slowed down due to edge friction while still gaining some speed from the pre-strafe. The shape of the platform plays a big role in High Jumps, a small square platform can't really be taken advantage of by this trick. 
```

Comment:
> so if you are moving in a direction that does not intersect with a platform's edge you are not considered "next to an edge" regardless of how close you actually are to it.

not quite right, you have to be moving in a direction where the edge check **touches** something, then you're **not** considered near an edge, otherwise if the edge check doesn't touch anything, you'll be considered near an edge.